var/spack/repos/builtin/packages/libceed/package.py

```diff
@@ -35,8 +37,9 @@ class Libceed(MakefilePackage, CudaPackage, ROCmPackage):
         depends_on('hip@3.8.0:', when='@0.8:')
         depends_on('hipblas@3.8.0:', when='@0.8:')
 
+    conflicts('+occa', when='@0.9:')
```

Comment:
I think I got this right. Will this also cover `@develop`?

Reply:
You can see the version ordering description in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison.

Reply:
You can see the version ordering description in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison.